### PR TITLE
Refactor Homepage UI

### PR DESCRIPTION
**Pull request by Daltz333**
Should fill out two more panels worth of space, or go into three rows

image

### CODE REVIEW

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:14]:
```suggestion
Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation! This site contains everything you need to know for programming a competition robot!
```

"will contain" is future tense.  I like using present here.  As is sounds like things are incomplete.

Comment (jasondaming) [inline source/index.rst:14]: not sure it contains everything needed for "building" a robot but programming we do pretty good.  So maybe cut building?

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:16]:
```suggestion
Community translations can be found in a variety of languages in the bottom-left menu.
```

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:12]: remove last 2 =

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:69]:
```suggestion
   An overview of the hardware components available to teams.
```

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:85]:
```suggestion
   An overview of the software components and tools available to teams.
```

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:122]:
```suggestion
   FRC Driver Station, provided by NI, is the tool used for enabling and controlling your robot.
```

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:137]: "Get introduced into" sounds awkward.

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:152]:
```suggestion
   The roboRIO, provided by NI, is the computer processor responsible for directly controlling your robot.
```

File: source/index.rst
Comment (jasondaming) [inline source/index.rst:167]:
```suggestion
   Command-Based programming is a framework that helps provide structure through subsytems and commands.
```

File: source/index.rst
Comment (sciencewhiz) [inline source/index.rst:29]: This may undersell what is needed for a returning team

File: source/index.rst
Comment (sciencewhiz) [inline source/index.rst:45]:
```suggestion
   Please read through the Zero-to-Robot tutorial! This will guide you through preparation, building and programming a basic robot!
```

File: source/index.rst
Comment (sciencewhiz) [inline source/index.rst:107]:
```suggestion
   Visual Studio Code is the official Integrated Development Environment for FRC\ |reg|. These documents highlight the usage necessary for teams.
```

File: source/index.rst
Comment (sciencewhiz) [reply source/index.rst:137]: components shouldn't be capitalized

File: source/index.rst
Comment (sciencewhiz) [inline source/index.rst:182]:
```suggestion
   Dashboards such as Shuffleboard and Glass are integral means of viewing diagnostic and other data from your robot.
```

File: source/index.rst
Comment (sciencewhiz) [inline source/index.rst:191]: Do we need the TOC now, or can we just refer people to the TOC on the left?

Comment (Daltz333) [reply source/index.rst:191]: We can't easily hide the ToC from the body without messing up navigation.

Comment (sciencewhiz) [reply source/index.rst:191]: It looks like there is a :hidden: directive that can be used to do this. https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#toctree-directive

File: source/index.rst
Comment (Daltz333) [reply source/index.rst:14]: We do have a build/assembly guide.

Comment (sciencewhiz) [reply source/index.rst:14]: The subtlety being that the guide is only for assembling the kitbot, which isn't a full competition robot.

File: source/stubs/api-docs-stub.rst
Comment (sciencewhiz) [inline source/stubs/api-docs-stub.rst:6]:
```suggestion
Java and C++ class documentation.
```

File: source/index.rst
Comment (sciencewhiz) [inline source/index.rst:213]: This also covers contributing to documentation

File: source/stubs/software-tools-stub.rst
Comment (sciencewhiz) [inline source/stubs/software-tools-stub.rst:6]: roboRIO Imaging tool actually isn't covered here. It's only in ZTR

**Review by jasondaming — APPROVED** [review]

**Comment by jasondaming** [comment]
Looks good but I think we need / want Kevin's approval before merge

**Comment by Kevin-OConnor** [comment]
The ZTR card talks about "building" your robot which is not something that these docs will actually cover, perhaps "wiring"? Other than that LGTM

**Review by Kevin-OConnor — APPROVED** [review]

**Pull request merged**
